src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,97 @@
+
+If you need to use image annotations:
+
+* Be concise.
+* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
+* Annotations aren't translated, so use them sparingly.
+* Don't use an annotation at the top that acts like a title. Use a caption.
+* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent. 
+
+  To install the New Relic SnagIt theme:
+    1. Ensure you're running the latest version of SnagIt.
+    2. In GitHub Desktop, clone the [repo](https://github.com/newrelic/newrelic-doc-tools) locally.
+    3. Open SnagIt and select any tool to open the right nav.
+    4. Under the **Quick Styles** pane in the right nav, select the **Theme** gear icon, then **Import**.
+
+## Callout boxes [#callout-boxes]
+
+Use callouts to highlight an element within a screenshot. If possible, don't add text to screenshots, that's what our docs are for. Callout boxes should not obstruct any of the elements they're calling out. For example, a callout box around a UI button shouldn't cover any part of the button. If possible, a 2-pixel clearance should be given between the UI element and the callout box.
+
+This is the format for adding callout boxes:
+  * Shape: square or rectangle, transparent fill, no border-radius.
+  * Border color: #00CE7C
+  * Line width: 5px
+  * Opacity: 100%
+  * Line style: solid
+  * No shadows
+
+<img
+  title="Example of a callout box"
+  alt="Example of a callout box"
+  src={styleguideExampleCalloutBox}
+/>
+
+<figcaption>
+  This is an example of a callout box.
+</figcaption>
+
+## Callout text [#callout-text]
+
+Screenshots shouldn't require additional text on them in 99% of all situations; it's the job of the written documentation itself to explain what is being represented in the screenshot. However, if a text box is required, it shouldn't overlap any essential elements of the UI.
+
+In case of adding text, follow this format:
+
+  * Shape: square or rectangle, fill color #00CE7C, no border-radius.
+  * Border color: #00CE7C
+  * Line width: 5px
+  * Opacity: 100%
+  * Line style: solid
+  * No shadows
+  * Font: Arial bold #FFFFFF, size 24px, left justified
+
+<img
+  title="dashboards_api_entity_section.png"
+  alt="dashboards_api_entity_section.png"
+  src={apisanddataNerdGraphDashboardEntity}
+/>
+
+<figcaption>
+  An example of a callout with text.
+</figcaption>
+
+## Arrows [#arrows]
+
+Use a single-headed arrow in your screenshot when you want to highlight an element within the UI. If you decide to add an arrow to a screenshot, try to not overlap any essential elements of the UI.
```

Comment:
@nbaenam what about arrows with tips at each end?

Reply:
I think we usually use single-headed arrows to highlight a specific part of the UI, but, maybe we can add that you can use a two-headed arrow if you need it. Is that what you mean? I'm not an expert here...

Reply:
In the previous theme, we had an arrow with arrowheads on each end. This was to show back-and-forth flow between items. I haven't used this much, but you might mention it.

Reply:
Yes, you're right. Added!